stereo_image_proc/src/nodelets/disparity.cpp

```diff
@@ -196,8 +242,32 @@ void DisparityNodelet::imageCb(const ImageConstPtr& l_image_msg,
   const cv::Mat_<uint8_t> l_image = cv_bridge::toCvShare(l_image_msg, sensor_msgs::image_encodings::MONO8)->image;
   const cv::Mat_<uint8_t> r_image = cv_bridge::toCvShare(r_image_msg, sensor_msgs::image_encodings::MONO8)->image;
 
+  // Downsampling
+  int downsampling_factor;
+  getPrivateNodeHandle().param("downsampling_factor", downsampling_factor, 1);
```

Comment:
Getting this parameter should be handled up in `onInit` as with other parameters.

Reply:
done